docs/blog/accessibility.md

```diff
@@ -0,0 +1,45 @@
+---
+title: The Accessibility of Lion
+published: true
+description: What does the term Accessibility mean within the context of Lion?
+date: 2021-12-07
+tags: [ux, accessibility]
+cover_image: /blog/images/accessibility.png
+---
+
+## Accessibility within Design Systems
+
+When I ([Erik Kroes](https://www.twitter.com/erikKroes)) worked together with developers all through ING, many of them pointed to the design system when it came to accessibility. It's easy to say something like: "we're doing it right, but it's the design system where the issues are!"
+
+### Design systems scale!
+
+To avoid such issues, it felt good to focus on the design system. It's a crucial point in the workflow of many developers and designers. No, an accessible design system does not warrant an accessible outcome. But yes, an inaccessible design system is pretty much assured to produce inaccessible products. It's hard to build something stable (a product) with an unstable foundation (an inaccessible design system). It's also a great way to improve for many by solving an issue once.
+Whatever you do well in a design system, will spread. But whatever you do wrong, also spreads. The sword is double-edged.
+
+### Quality, quality and quality
+
+If the previous point isn't a good reason to focus on quality by itself, there's another. Design systems receive the scrutiny of all the people that work with them. When person Y thinks the performance of component Z could be improved, it might mean the entire team won't use your design system. Then comes person X who sees improvements for component N. And with every negative impression, people will trust the entire system less. A chain is no stronger than its weakest link, and the impression of a design system is based on its worst component.
+The challenge is to let quality be a strength instead of a weakness. The challenge is to include the people that work with your design system. Incude and have them contribute their expertise. Imagine person Y applied their expertise on performance to help improve. That would be a benefit for person X, who could contribute their expertise.
```

Comment:
```suggestion
The challenge is to let quality be a strength instead of a weakness. The challenge is to include the people that work with your design system. Include and have them contribute their expertise. Imagine person Y applied their expertise on performance to help improve. That would be a benefit for person X, who could contribute their expertise.
```